__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const { app } = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const database = require("../db/connection");
+const testData = require("../db/data/test-data/index");
+const { response } = require("express");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  database.end();
+});
+
+describe("nc-news", () => {
+  describe("GET /api/topics", () => {
+    test("Returns status 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("responds with an array of topic objects", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const allTopics = response.body.topics;
+          expect(Array.isArray(allTopics)).toBe(true);
+          allTopics.forEach((topic) => {
```

Comment:
Make sure you check for length on allTopics. If the array is not populated, your for each with give a false positive test.